stjude/proteinpaint
# Burden routes

## Description
This creates a server route for cumulative burden estimates. Pull the dataset burden updates and required input RData files per https://github.com/stjude/sjpp/pull/167.

To test: 
1. Should see a `status: "ok"` with the [default burden route URL parameters](http://localhost:3000/burden?genome=hg38&dslabel=SJLife&diaggrp=5&sex=1&white=1&agedx=1&bleo=0&etop=0&cisp=0&carbo=0&steriod=0&vcr=0&hdmtx=0&itmt=0&ced=0&dox=0&heart=0&brain=0&abd=0&pelvis=0&chest=0). (take about 10 seconds - to be optimized)
2. Can also test with `cd server; npm run test:type`

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
